Cargo.lock

```diff
@@ -3555,6 +3586,9 @@ dependencies = [
 "checksum error-chain 0.11.0 (registry+https://github.com/rust-lang/crates.io-index)" = "ff511d5dc435d703f4971bc399647c9bc38e20cb41452e3b9feb4765419ed3f3"
 "checksum eth-secp256k1 0.5.6 (git+https://github.com/paritytech/rust-secp256k1)" = "<none>"
 "checksum ethabi 4.0.1 (registry+https://github.com/rust-lang/crates.io-index)" = "c819a3adef0413a2519cbd9a19a35dd1c20c7a0110705beaba8aa4aa87eda95f"
+"checksum ethabi 4.1.0 (git+https://github.com/paritytech/ethabi/?branch=call-function)" = "<none>"
+"checksum ethabi-contract 4.1.0 (git+https://github.com/paritytech/ethabi/?branch=call-function)" = "<none>"
+"checksum ethabi-derive 4.1.0 (git+https://github.com/paritytech/ethabi/?branch=call-function)" = "<none>"
```

Comment:
Will checksums automatically get added to the ethabi dependencies? I'm not very familiar with how checksums are calculated/stored for github dependencies.

Reply:
I don't know, I'm not familiar with them either, but we will switch to a regular dependency (on crates.io) once the ethabi branch is merged; this will probably add the checksums again?

Reply:
Yeah, pretty sure that crates.io includes the checksum with the crate. I'll remove this change-request.

Reply:
I'm waiting for https://github.com/paritytech/ethabi/pull/62 to be merged and then I'll update the dependency in this PR.

Reply:
Ok sounds good. Everything else looks great. So far from the tests I've written, your changes are easy to incorporate.

Reply:
I've added `paritytech/core-devs` to the owners list of [ethabi_derive](https://crates.io/crates/ethabi_derive). I've also released 4.2, so feel free to use it instead of github repo ;)